ndex2/client.py

```diff
@@ -1090,7 +1627,25 @@ def create_networkset(self, name, description):
     def get_network_set(self, set_id):
         """
         Gets the network set information including the list of networks
+	
+	.. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
 
+		set_id = 'INSERT SET UUID HERE'
+
+		#this method returns a dict of the set information
+		network_set = my_ndex.get_network_set(set_id)
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client